examples/kuesa/tonemapping/MainMenu.qml

```diff
@@ -0,0 +1,273 @@
+/*
+    MainMenu.qml
+
+    This file is part of Kuesa.
+
+    Copyright (C) 2019 Klarälvdalens Datakonsult AB, a KDAB Group company, info@kdab.com
+    Author: Mike Krus <mike.krus@kdab.com>
```

Comment:
Maybe you want to fix attribution

Reply:
Whoops, didn't notice that line in the header. Done now.